static/js/tools/map/chart_loader.tsx

```diff
@@ -134,6 +141,46 @@ export function ChartLoader(): JSX.Element {
   );
 }
 
+function getGeoJsonDataFeatures(
+  placeDcids: string[],
+  enclosedPlaceType: string
+): GeoJsonFeature[] {
+  const distance = MANUAL_GEOJSON_DISTANCES[enclosedPlaceType];
+  if (!distance) {
+    return [];
+  }
+  const geoJsonFeatures = [];
+  for (const placeDcid of placeDcids) {
+    if (!placeDcid.includes("/") || !placeDcid.includes("_")) {
+      continue;
+    }
+    const latlon = placeDcid.split("/")[1].split("_");
+    const neLat = Number(latlon[0]) + distance / 2;
+    const neLon = Number(latlon[1]) + distance / 2;
+    const placeName = `${latlon[0]}, ${latlon[1]} (${distance} resolution)`;
+    geoJsonFeatures.push({
```

Comment:
Not sure if the drawing libraries handle overflowing 180 near the intl date line. Add a TODO to handle those cases?

Reply:
done